README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# SUBSCRIPTIONS CUSTOMER PORTAL
 
```

Comment:
Consider adding a description here that explains the purpose of this project.

Something like: 

```
## Description

This open source project serves as a starting point for developers creating a customer portal from scratch that is integrated with Bold Subscriptions.
```